starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -50,3 +50,11 @@ public void findAll() {
     Assert.assertFalse(actualInventory.isEmpty());
   }
 }
+//task1
+public void createDAOtest() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    this.InventoryDAO.create(inventory);
```

Comment:
We want the inventory returned from `this.InventoryDAO.create(inventory);` to test.
Have an object created from `this.InventoryDAO.create(inventory);`